cmd/gateway-gcs.go

```diff
@@ -248,11 +252,36 @@ type gcsGateway struct {
 
 const googleStorageEndpoint = "storage.googleapis.com"
 
+// Returns projectID from the GOOGLE_APPLICATION_CREDENTIALS file.
+func gcsParseProjectID(path string) (projectID string, err error) {
```

Comment:
`credsJSONFile` would be more clear than `path`

Reply:
I have renamed it to `credsFile`